### ISSUE TITLE
details/summary has no visible focus style.

### ISSUE THREAD

**Issue by fuzzbomb** [thememachine/eleven#160]
# The issue is:
The `<summary` elements are focusable and operable, but lack a visible focus style.  A sighted keyboard relies on these to know when they can press spacebar or enter to open/close the details element.

When a set of adjacent details are all closed, it's hard to be sure which one is focused, and a user is likely to open the wrong one unless they count TAB presses carefully.

Also the `[+] Show info` summary elements have a different style;  these likewise lack a visible focus indicator. 

# Where
- `admin/modules` is a good best place to see this; it has both styles of summary elements
- `node/add/page` shows the problem with adjacent closed details groups.  Click in the "revision log message" text area, then press TAB a few times.  You'd expect to see the menu/path/author/promotion summary elements come into focus with each press.  Press space, and you might be surprised which one opens.

# Todo
- Give it a clear visible focus style, using colours that pass WCAG 1.4.3 contrast minimum. 
- Don't rely on colour alone - it would be good to use a focus outline, or animate the icon, or some other shape change.
- The smaller "show info" version currently looks more like a link than a button, but the behaviour of summary elements is like a button (buttons and summary respond to spacebar OR enter keys when focused; links just respond to enter).  It might be tricky to find an attractive focus style for this.

**Comment by mortendk** [comment]
![image](https://user-images.githubusercontent.com/65756/29069213-34f4574e-7c3a-11e7-8ee9-f52de5199ad2.png)

![image](https://user-images.githubusercontent.com/65756/29069220-3cbe16a4-7c3a-11e7-979e-ce50caa15f8b.png)

**Comment by fuzzbomb** [comment]
Cool, very clear focus on both kinds of details/summary
